Src/IronPython/Runtime/WeakRef.cs

```diff
@@ -91,67 +94,133 @@ struct CallbackInfo {
         }
 
         public void ChainCallback(object callback, object weakRef) {
-            _callbacks.Add(new CallbackInfo(callback, weakRef));
+            _lock.EnterWriteLock();
+            try
```

Comment:
Can you reformat so that the braces are on the same lines as the try and finally? This is the style we use in the code.

Reply:
No probs - I'll put in another commit for this

Reply:
ok - I've updated the styles - can you check that it looks ok now?